<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(190, 190, 190);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,4">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>30</y>
           <width>221</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>BankGothic Lt BT</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Aplicaciones</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QPushButton" name="Boton_Ajedrez">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>131</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>BankGothic Lt BT</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(8, 121, 197);</string>
         </property>
         <property name="text">
          <string>Ajedrez</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>LOGO_AJEDREZ.png</normaloff>LOGO_AJEDREZ.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="Boton_Gato">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>30</y>
           <width>131</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>BankGothic Lt BT</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 126, 14);</string>
         </property>
         <property name="text">
          <string>Gato</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>logo_Gato4.png</normaloff>logo_Gato4.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="Boton_Batimovil">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>30</y>
           <width>131</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>BankGothic Lt BT</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(50, 50, 50);</string>
         </property>
         <property name="text">
          <string>Batimovil</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>logoBtmn.png</normaloff>logoBtmn.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="Boton_Regresar">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>210</y>
           <width>131</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>BankGothic Lt BT</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 79, 10);</string>
         </property>
         <property name="text">
          <string>Regresar</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditUndo"/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
